### DISCUSSION TITLE
add github icon on site

### DISCUSSION THREAD

**Discussion by Arhell** [npm/feedback#93] [Website] [answered]
Good afternoon, but there is an opportunity to add a social icon of the github (place of organization) to the footer, I just didn't find it on the site.
As for me it will be very convenient.
I can do it if you will.

**Comment by MylesBorins** [comment]
Hey @Arhell 

Are you asking for this on npmjs.com? Are you imagining this would forward to GitHub.com?

**Reply by ethomson** [reply]
Yeah, I'm curious, too.  Something like this?

![Screen Shot 2020-11-13 at 15 06 58](https://user-images.githubusercontent.com/1130014/99088868-83cd0000-25c4-11eb-9788-a9296dfb077c.png)

**Reply by Arhell** [reply]
@MylesBorins yes
@ethomson option. but it is not necessary to dwell on this, you can add others, twitter and others if any)

here is my version
![0e84230d61](https://user-images.githubusercontent.com/26163841/99101568-71af8980-25e5-11eb-96a0-0d0c37354846.jpg)

**Comment by Arhell** [comment, marked as answer]
@MylesBorins, can i help with adding icons?

**Reply by MylesBorins** [reply]
Hey @Arhell thanks for reaching out.

The npm website is not open sourced and as such you will unfortunately not be able to help with adding these icons. We do have another similar issue on the roadmap for this quarter about making the URL for a repo easier to discover, when we get that work lined up we can see if this is something we can include in the work item

https://github.com/npm/roadmap/issues/8

**Comment by NatashaDorfman** [comment]
It was implemented 
![Screen Shot 2021-07-18 at 14 03 57](https://user-images.githubusercontent.com/68326101/126064768-f8963d10-d566-4d73-aef4-8e9063530b38.png)

**Reply by Arhell** [reply]
Thanks, but it would be better if a link was added there and not just a picture https://github.com/npm or https://github.com/npm/cli

**Reply by MylesBorins** [reply]
@Arhell I think if it were a link it would likely go to https://github.com rather than a specific npm repo or the org

**Reply by restrictedCodex** [reply]
@MylesBorins it would be better if it redirects to npm's official GitHub page at  https://github.com/npm

**Reply by MylesBorins** [reply]
We've updated the icon to be a link PTAL
